src/main/kotlin/racingcar/Cars.kt

```diff
@@ -0,0 +1,23 @@
+package racingcar
+
+class Cars {
+    private var _list = mutableListOf<Car>()
+    val list: List<Car>
+        get() = _list
+
+    fun addAllList(list: List<Car>) {
+        list.forEach { car ->
+            _list.add(car)
+        }
+    }
```

Comment:
```suggestion
    fun addAllList(list: List<Car>) = _list.addAll(list)
```
`addAll()` 이라는 선택지도 있습니다!

Reply:
앗,,, 엄청 간결해졌네요,,!! 감사합니다😀